### DISCUSSION TITLE
Compatibility with Logic Pro 9?

### DISCUSSION THREAD

**Discussion by nobonesrecords** [apohl79/audiogridder#381] [Q&A]
Can anyone tell me if AudioGridder should be working in Logic Pro 9 on an Intel Mac Pro (2008) running OS 10.11.6? The plug-in is not showing up in my AudioUnits Manager.

**Comment by apohl79** [comment]
If you run auval (in a terminal), it should look like this:

```
$ auval -al 2>&1 | grep e47
aufx E401 XE47  -  e47: AGridder     (file:///Library/Audio/Plug-Ins/Components/AudioGridder.component/)
aumu E402 XE47  -  e47: AGridder (inst)     (file:///Library/Audio/Plug-Ins/Components/AudioGridderInst.component/)
aumi E403 XE47  -  e47: AGridder (midi)     (file:///Library/Audio/Plug-Ins/Components/AudioGridderMidi.component/)
```

If thats not the case, you can try to wipe the AU cache:

```
rm -rf ~/Library/Caches/AudioUnitCache/
```

Also, try to reboot.

**Reply by nobonesrecords** [reply]
Forgive me, but I'm an end-user/musician who knows nothing about coding and never uses Terminal! Can you clarify - would you expect AudioGridder to be working in Logic 9? If so, is there anything I can try so that it shows up in the AU Manager?

**Reply by apohl79** [reply]
The command above will show, if the plugin is available to logic. I can't tell you, if it works with logic 9, as I don't have access to that version.

So open the terminal app (under Applications/Utilities) and copy "auval -al 2>&1 | grep e47" into it and hit enter. If you don't see the output as shown above, run the second command (again copy and hit enter). Reboot. Try the first command again.

AG should - in theory - be backwards compatible to 10.7 (server and plugin), so your 10.11 system should be supported.

**Reply by nobonesrecords** [reply]
OK - thanks. Will give it a go...

**Reply by nobonesrecords** [reply]
Hi. I ran the commands as suggested - nothing! (Also re-booted). Screenshot of the Terminal display attached. Does this mean it is totally incompatible with Logic 9, in your view?
![terminal reading](https://user-images.githubusercontent.com/78929883/108930138-40f7db00-763d-11eb-94a9-d84ac9f8b9d1.jpg)

**Comment by apohl79** [comment]
Can you just run “auval -a“?

The problem is, if AG does not get listed, it will not be available to logic and I guess any other AU host application.

Do you see any logfiles when running “ls -l ~/Library/Logs/AudioGridder“ ?

**Reply by nobonesrecords** [reply]
I ran "auval -a" and got a long list, and at the end it reads:
"Cannot open component: -10847
aumi E403 XE47  -  e47: AGridder (midi)
diskStreamCallback exiting
diskStreamCallback exiting
diskStreamCallback exiting"

**Reply by nobonesrecords** [reply]
I also see this in the output stream:
"aumu E402 XE47  -  e47: AGridder (inst)"

**Reply by nobonesrecords** [reply]
This is what I get when I run the second command:
ls -l ~/Library/Logs/AudioGridder
total 0
drwxr-xr-x  5 dominicglynn  501  170 24 Feb 21:58 FX
drwxr-xr-x  4 dominicglynn  501  136 24 Feb 22:00 Instrument
drwxr-xr-x  4 dominicglynn  501  136 24 Feb 22:01 Midi

**Reply by apohl79** [reply]
That means something goes wrong when the plugins get scanned. Can you zip the log folder and attach it here?

Open finder, press command+shift+g, enter "~/Library/Logs/" and hit return. Now right click on AudioGridder and "Compress".

**Comment by nobonesrecords** [comment]
Hey. Log file attached…







> On 24 Feb 2021, at 22:27, Andreas Pohl <notifications@github.com> wrote:
> 
> ~/Library/Logs/

Dominic Glynn
 Composer of music for film & television	
 <https://www.facebook.com/DominicGlynn>	 <https://www.instagram.com/dominic_glynn/>	 <https://twitter.com/DominicGlynn1>	 <https://www.youtube.com/c/DominicGlynnMusic>
email: music@nobones.co.uk
Soundcloud: https://soundcloud.com/nobones-2

**Reply by apohl79** [reply]
No attachment. I think you can’t attach files via email.

**Reply by nobonesrecords** [reply]
[AudioGridder.zip](https://github.com/apohl79/audiogridder/files/6046729/AudioGridder.zip)
How's this?

**Reply by nobonesrecords** [reply]
Did you receive the attachment this time?

**Reply by apohl79** [reply]
Yes, that worked. Unfortunately, there is no errors in the logs, so the scans were successful. I don't know, whats wrong on your machine tbh. You could try another DAW with AU support, just to see if it would work with AudioGridder, but as auval is not showing it, I doubt it would work. 

I hope you get it working!

**Comment by prjkt** [comment]
Are you running Logic in 32bit mode? Highlight Logic's icon in /Applications and hit CMD+I to check.

If you are, Audiogridder won't appear

![Screen Shot 2021-03-13 at 7 40 32 PM](https://user-images.githubusercontent.com/67309493/111024668-1cb93e00-8434-11eb-9d50-91c76fc98547.png)

**Reply by nobonesrecords** [reply]
Thanks, but I am running Logic in 64Bit mode - but it's still not showing up.

**Comment by CaBuffalo** [comment]
AG doesn't support 32bit OS. It's 64 bit only.

On Sat, Mar 13, 2021 at 12:42 AM prjkt ***@***.***> wrote:

> Are you running Logic in 32bit mode? Highlight Logic's icon in
> /Applications and hit CMD+I to check.
>
> If you are, Audiogridder won't appear
>
> [image: Screen Shot 2021-03-13 at 7 40 32 PM]
> <https://user-images.githubusercontent.com/67309493/111024668-1cb93e00-8434-11eb-9d50-91c76fc98547.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/381#discussioncomment-477365>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6F77CPT3VZCTJBW6YLTDMQO7ANCNFSM4XPMEPMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by nobonesrecords** [reply]
Absolutely - but it doesn't appear to be working in Logic 9 at 64 bit.
